File: SPOJ/4200.c
#include<stdio.h>
#include<cmath>
int main()
{
	int test_cases;
	scanf("%d",&test_cases);
	while(test_cases--)
	{
		double ans,theta,x,y,v,k1,k2,pi=2*acos(0);
		scanf("%lf%lf%lf",&v,&k1,&k2);
		x=-1.0*4.0*(k1/k2);
		theta = pi/2+atan(x)/2;
		ans = (2*k1*v*v*sin(theta)*cos(theta))/10 + (k2*v*v*sin(theta)*sin(theta))/20;
		printf("%.3lf %.3lf\n",theta,ans);
	}
	return 0;
}

File: SPOJ/4200.py
from math import *
t=input()
while t:
	t=t-1
	x=raw_input().split()
	v=int(x[0])
	k1=int(x[1])
	k2=int(x[2])
	pi=2*acos(0.0)
	if k2==0:
		theta = pi/4
	else:
		theta = (pi+atan((-4.0*k1)/k2))/2
	ans = (((v*v)/(10))*(k1*sin(2.0*theta)))+(((v*v)/10)*((k2*sin(theta)*sin(theta))/2))
	print round(theta,3),round(ans,3)
